tests/sentry/api/endpoints/test_organization_index.py

```diff
@@ -236,9 +234,9 @@ def test_name_slugify(self, is_limited):
         assert org.slug == "canada"
         assert org_slug_pattern.match(org.slug)
 
-        response = self.get_success_response(name="1234")
+        response = self.get_success_response(name="1234-foo")
         org = Organization.objects.get(id=response.data["id"])
-        assert org.slug == "1234"
+        assert org.slug == "1234-foo"
```

Comment:
Like `test_slugs` above, this was originally passing because we forgot to put the option on this test.